eng/pipelines/common/evaluate-changed-paths.yml

```diff
@@ -0,0 +1,23 @@
+# This file is intended to evaluate git changes using git based on a include/exclude path filter.
+# For more information on how the path evaluation works look at evaluate-changed-paths.sh docs
+# at the beginning of that file.
+#
+# paramters
+# subsetName:
+#   Name for the subset that we're evaluating changes for.
```

Comment:
I prefer parameter docs on the line above each parameter for maintainability and automatically seeing the expected type from the default value.

Reply:
Makes sense. Will do that.